.github/workflows/review.yml

```
@@ -14,6 +14,9 @@ jobs:
     if: ${{ github.event.label.name == 'review' }}
     name: Deploy Storybook to GH Pages
     runs-on: ubuntu-latest
+    environment:
+      name: github-pages
+      url: ${{steps.deployment.outputs.page_url}}
```

actionlint (1.7.4)

property "deployment" is not defined in object type {deploy: {conclusion: string; outcome: string; outputs: {page_url: string}}}

(expression)